css/index.css

```diff
@@ -172,4 +172,198 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
+}
+
+/* ADAPTIVE STYLES BELOW: */
+
+@media (max-width: 1100px) {
+
+    header nav ul {
+        margin-top: 40px;
+        width: 450px;
+        justify-content: space-around;
+        display: flex;
+    }
+
+    .cta .cta-text {
+        font-size: 72px;
+        text-align: center;
+        letter-spacing: 5px;
+        display: flex;
+        justify-content: center;
+        flex-flow: column;
+        align-items: center;
+        margin: 0px 40px 0 0;
+    }
+
+    .cta img {
+        margin-right: 150px;
+    }
+
+    .main-content {
+        border: none;
+    }
+
+    .main-content .middle-img {
+        display: none;
+    }
+
+    .main-content .top-content {
+        display: flex;
+        flex-direction: column;
+        width: 700px;
+        margin: 0 0 50px 0;
+        padding: 50px;
+    }
+
+    .main-content .bottom-content {
+        display: flex;
+        flex-direction: column;
+        width: 700px;
+        padding: 50px;
+    }
+
+    .contact {
+        margin: 15px 55px;
+    }
+
+    footer {
+        margin-right: 100px;
+    }
+}
+
+@media (max-width: 768px) {
+
+    header nav ul {
+        margin-top: 10px;
+        padding-left: 50px;
+        width: 150px;
+        justify-content: space-around;
+        display: flex;
+        flex-direction: column;
+
```

Comment:
You can add `overflow: hidden;` to remove the scrollbar at the bottom of this and the smaller breakpoint.